packages/renderer/tests/mock-backend/README.md

```
@@ -0,0 +1,39 @@
+
+\`\`\`bash
+cd packages/renderer/tests/mock-backend
+npm install
+npm start
+\`\`\`
+
+### Testing Connection
+
+\`\`\`javascript
+// In browser console
+const socket = io('http://localhost:8081');
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)